examples/reference_plants/05-offshore-hydrogen-ca/input/plant/greenheart_config_offshore_ca.yaml

```diff
@@ -81,10 +81,10 @@ h2_transport_pipe:
   outlet_pressure: 10 # bar - from example in code from Jamie #TODO check this value
 h2_storage:
   size_capacity_from_demand:
-    flag: False # If True, then storage is sized to provide steady-state storage    
+    flag: True # If True, then storage is sized to provide steady-state storage    
   capacity_from_max_on_turbine_storage: False # if True, then days of storage is ignored and storage capacity is based on how much h2 storage fits on the turbines in the plant using Kottenstete 2003.
-  type: "none" # can be one of ["none", "pipe", "turbine", "pressure_vessel", "salt_cavern", "lined_rock_cavern"]  
-  days: 3 # [days] how many days worth of production we should be able to store (this is ignored if `capacity_from_max_on_turbine_storage` is set to True)
+  type: "lined_rock_cavern" # can be one of ["none", "pipe", "turbine", "pressure_vessel", "salt_cavern", "lined_rock_cavern"]  
+  days: 0 # [days] how many days worth of production we should be able to store (this is ignored if `capacity_from_max_on_turbine_storage` is set to True)
```

Comment:
Why do some of the yamls have `day: -1` and some have `days: 0`? Do both have the same effect? I think we should be consistent here.